dts/msm8974/msm8974pro-sony-xperia-shinano-leo-mdss.dtsi

```diff
@@ -0,0 +1,1340 @@
+/ {
```

Comment:
Since you import this, can you also inherit the copyright boilerplate like in https://github.com/msm8916-mainline/lk2nd/blob/master/dts/msm8916/motorola-carrier-channel-ids.dtsi

Reply:
done